### ISSUE TITLE
Segfault in SkiaEncoder

### ISSUE THREAD

**Issue by cvium** [jellyfin/jellyfin#1298]
**Describe the bug**
If the path supplied to GetImageSize doesn't exist, it segfaults at https://github.com/jellyfin/jellyfin/blob/master/Jellyfin.Drawing.Skia/SkiaEncoder.cs#L189 (at least on macOS).

**System (please complete the following information):**
 - OS: macOS
 - Browser: Irrelevant
 - Jellyfin Version: 10.2+
 - Reverse proxy: no

**Comment by PrplHaz4** [comment]
I think this was also happening on my windows installation every time I started jellyfin, it would crash.
```
Faulting application name: jellyfin.exe, version: 0.0.0.0, time stamp: 0x5c8ad626
Faulting module name: libSkiaSharp.DLL, version: 0.0.0.0, time stamp: 0x5c0356e7
Exception code: 0xc0000409
Fault offset: 0x0000000000460293
Faulting process id: 0x37a4
Faulting application start time: 0x01d51bdbb9a4fdf6
Faulting application path: C:\apps\jellyfin_10.3.3\jellyfin.exe
Faulting module path: C:\apps\jellyfin_10.3.3\libSkiaSharp.DLL
Report Id: 56383e28-0f6e-4e2b-859a-0e909efc5f41
Faulting package full name: 
Faulting package-relative application ID: 
```
```
Faulting application name: jellyfin.exe, version: 0.0.0.0, time stamp: 0x5c8ad626
Faulting module name: libSkiaSharp.DLL, version: 0.0.0.0, time stamp: 0x5c0356e7
Exception code: 0xc0000409
Fault offset: 0x0000000000460293
Faulting process id: 0x37a4
Faulting application start time: 0x01d51bdbb9a4fdf6
Faulting application path: C:\apps\jellyfin_10.3.3\jellyfin.exe
Faulting module path: C:\apps\jellyfin_10.3.3\libSkiaSharp.DLL
Report Id: 56383e28-0f6e-4e2b-859a-0e909efc5f41
Faulting package full name: 
Faulting package-relative application ID: 
```

**Comment by cvium** [comment]
Will be fixed in 10.3.6. Closing for now.
